### ISSUE TITLE
Better to have slides on proper archival site, get DOI, and just list here

### ISSUE THREAD

**Issue by labarba** [deniederhut/Slides-SciPyConf-2018#18]
It may be too late for this year, but let me say that I don't think posting all the slides in a GitHub repo is a great idea. It would be much better for authors to deposit their slides in a proper archival site, get a DOI, and use the GitHub repo to simply _list_ the presentations, with complete citation including their DOI.

I, for one, prefer not to add my slides on this repo, as I already have uploaded them to Figshare, so the citation is:

> Barba, Lorena A. (2018): Engineers Code: re-usable, open educational modules for engineering undergraduates. figshare. Presentation. https://doi.org/10.6084/m9.figshare.6818279.v1

**Comment by leouieda** [comment]
@labarba I agree and I did the same. I also noticed that a few of the posters in the README are DOI links to zenodo and F1000, so clearly some people are already doing this and got merged. 

Do you think linking using the DOI is enough or would it have to be a full citation?

**Comment by labarba** [comment]
It's always a good idea to remind people how you want your work to be cited (many people don't know!).
